### ISSUE TITLE
An obsession

### ISSUE THREAD

**Issue by 731076467** [SvipRepetitionCounting/TransRAC#19]
I always have the problem of "type float can't be cast to the desired output type long" after training several epoch. Sometimes the second epoch reports an error, and sometimes it reports an error on the ninth epoch.

**Comment by isotopezzq** [comment]
I also run into this problem and after 3 epochs the traing stopped

**Comment by Ironieser** [comment]
Sry, I tried to reproduce your bug today. But I failed. Everything is OK in my experiment.  
At least, in my training, the codes have no any bugs and I used the processed npz as input  data rather than raw videos.
Maybe you should check your env and the input data.
The training log as the following figure shown.
![Snipaste_2022-07-28_19-42-57](https://user-images.githubusercontent.com/32543247/181497085-41d934c4-b84f-4a0d-92e4-323ce54203a5.png)

**Comment by Ironieser** [comment]
If you need our help, please upload more details of your env and log. :)

**Comment by 731076467** [comment]
The version of pytorch I use is 1.8.0+cu111, and my device only has a 3080-10G GPU, so the batch size for training is relatively small. Is it possible that this is the problem?

**Comment by Ironieser** [comment]
> The version of pytorch I use is 1.8.0+cu111, and my device only has a 3080-10G GPU, so the batch size for training is relatively small. Is it possible that this is the problem?

No, there is no problem. Maybe others

**Comment by isotopezzq** [comment]
> > The version of pytorch I use is 1.8.0+cu111, and my device only has a 3080-10G GPU, so the batch size for training is relatively small. Is it possible that this is the problem?
> 
> No, there is no problem. Maybe others
Is there any chance that it would be the problem of dataset? (I noticed that in stu6_44, the label was lost)
![image](https://user-images.githubusercontent.com/63627336/181508233-bb504d98-c310-430e-9fee-fdbd3d35b188.png)

**Comment by isotopezzq** [comment]
I delete the missing data(stu6_44.mp4),and everything goes well

**Comment by 731076467** [comment]
Which file is this file?

**Comment by isotopezzq** [comment]
in train.csv line 585
delete this line and also delete the video in ./video/train

**Comment by 731076467** [comment]
thank you!let me try!

**Comment by Ironieser** [comment]
> > > The version of pytorch I use is 1.8.0+cu111, and my device only has a 3080-10G GPU, so the batch size for training is relatively small. Is it possible that this is the problem?
> > 
> > 
> > No, there is no problem. Maybe others
> > Is there any chance that it would be the problem of dataset? (I noticed that in stu6_44, the label was lost)
> > ![image](https://user-images.githubusercontent.com/63627336/181508233-bb504d98-c310-430e-9fee-fdbd3d35b188.png)

yeah, I think so.

**Comment by Wantong666** [comment]
但是我删除了585，还是在第9个epoch中断了，是怎么回事呢？
